ioctl/cmd/node/nodedelegate.go

```diff
@@ -101,43 +105,15 @@ func (m *delegatesMessage) String() string {
 		}
 		lines := []string{fmt.Sprintf("Epoch: %d,  Start block height: %d,Total blocks produced in epoch: %d\n",
 			m.Epoch, m.StartBlock, m.TotalBlocks)}
-		formatTitleString := "%-41s   %-4s   %-" + strconv.Itoa(aliasLen) + "s   %-6s   %-6s   %-12s    %s"
-		formatDataString := "%-41s   %4d   %-" + strconv.Itoa(aliasLen) + "s   %-6s   %-6d   %-12s    %s"
+		formatTitleString := "%-41s   %-12s   %-4s   %-" + strconv.Itoa(aliasLen) + "s   %-6s   %-6s   %-12s    %s"
+		formatDataString := "%-41s   %-12s   %4d   %-" + strconv.Itoa(aliasLen) + "s   %-6s   %-6d   %-12s    %s"
 		lines = append(lines, fmt.Sprintf(formatTitleString,
-			"Address", "Rank", "Alias", "Status", "Blocks", "ProbatedStatus", "Votes"))
-		for _, bp := range m.Delegates {
-			lines = append(lines, fmt.Sprintf(formatDataString, bp.Address, bp.Rank,
-				bp.Alias, nodeStatus[bp.Active], bp.Production, probatedStatus[bp.ProbatedStatus], bp.Votes))
-		}
-		return strings.Join(lines, "\n")
-	}
-	return output.FormatString(output.Result, m)
-}
-
-type nextDelegatesMessage struct {
-	Epoch      int        `json:"epoch"`
-	Determined bool       `json:"determined"`
-	Delegates  []delegate `json:"delegates"`
-}
-
-func (m *nextDelegatesMessage) String() string {
-	if output.Format == "" {
-		if !m.Determined {
-			return fmt.Sprintf("delegates of upcoming epoch #%d are not determined", epochNum)
-		}
-		aliasLen := 5
-		for _, bp := range m.Delegates {
-			if len(bp.Alias) > aliasLen {
-				aliasLen = len(bp.Alias)
+			"Address", "Name", "Rank", "Alias", "Status", "Blocks", "ProbatedStatus", "Votes"))
+		for i, bp := range m.Delegates {
+			if i == 36 {
+				lines = append(lines, "------------------------------")
```

Comment:
probably just remove this. this make an assumption that 36 is an important number and hard coded here

Reply:
we cannot make this assumption that top 36 is different than others. so this separate line is not necessary

Reply:
I'll remove it

Reply:
add this after active producer list

Reply:
In String(),it's diffuclt to check which one is bp or not